src/build/sign.rs

```diff
@@ -1,30 +1,48 @@
 use std::fs;
 use std::fs::DirEntry;
-use std::io::{Error};
+use std::io::Error;
 use std::path::PathBuf;
 
-use crate::error::*;
-
-pub fn copy_sign(folder: &String, entry: &DirEntry, p: &crate::project::Project, version: &String) -> Result<bool, Error> {
+pub fn copy_sign(
+    folder: &String,
+    entry: &DirEntry,
+    p: &crate::project::Project,
+    version: &String,
+    key: &armake2::sign::BIPrivateKey,
+) -> Result<bool, Error> {
     let path = entry.path();
     let cpath = path.clone();
-    let cpath = cpath.to_str().unwrap().replace(r#"\"#,"/");
-    let pbo = cpath.replace((folder.clone() + "/").as_str(), "");
-    if !path.ends_with(".pbo") && !pbo.contains(p.prefix.as_str()) {
+    let cpath = cpath.to_str().unwrap().replace(r#"\"#, "/");
+    let pbo_path = cpath.replace((folder.clone() + "/").as_str(), "");
+    if !path.ends_with(".pbo") && !pbo_path.contains(p.prefix.as_str()) {
         return Ok(false);
     }
 
     let modname = p.get_modname();
-    fs::copy(&cpath, format!("releases/{}/@{}/{}/{}", version, modname, folder, pbo))?;
+    fs::copy(
+        &cpath,
+        format!("releases/{}/@{}/{}/{}", version, modname, folder, pbo_path),
```

Comment:
:thinking:

Reply:
What's the `thunk` for? `rustfmt` decided on the newline, not me :(

Reply:
It's hideous. :D